wd_helper.au3

```diff
@@ -686,8 +686,8 @@ EndFunc   ;==>_WD_HighlightElement
 ;                  1 - Highlight border dotted red
 ;                  2 - Highlight yellow rounded box
 ;                  3 - Highlight yellow rounded box + border  dotted red
```

Comment:
One whitespace is enough 😅 . Same for the function above `_WD_HighlightElement`.

Reply:
Not sure why it appears like this. They both appear the same on my end, and I didn't change anything in this area AFAIK